.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/psf/black
-    rev: 21.5b0
+    rev: 22.3.0
```

Comment:
Black 22.3.0 fixed an issue with `>8.1.0` versions of click that comes with black https://github.com/psf/black/issues/2964